roamance/src/hooks/useToggle.ts

```
@@ -0,0 +1,13 @@
+import { useState, useCallback } from 'react';
+
+export function useToggle(
+  initialState: boolean = false
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)